README.md

```diff
@@ -1,69 +1,70 @@
-# Go SafeCast
+# 🪄 go-safecast: safe numbers conversion
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/ccoveille/go-safecast)](https://goreportcard.com/report/github.com/ccoveille/go-safecast)
 [![GoDoc](https://godoc.org/github.com/ccoVeille/go-safecast?status.svg)](https://godoc.org/github.com/ccoVeille/go-safecast)
 [![codecov](https://codecov.io/gh/ccoVeille/go-safecast/graph/badge.svg?token=VW0VO503U6)](https://codecov.io/gh/ccoVeille/go-safecast)
 [![Code Climate](https://codeclimate.com/github/ccoVeille/go-safecast.png)](https://codeclimate.com/github/ccoVeille/go-safecast)
 
-## Origin of this project
+## Project purpose
 
-In Go, integer type conversion can lead to unexpected behavior and errors if not handled carefully.
+In Go, integer type conversion can lead to a silent and unexpected behavior and errors if not handled carefully.
 
-Issues can happen when converting between signed and unsigned integers, or when converting to a smaller integer type.
-
-The gosec project raised this to my attention when the gosec [G115 rule was added](https://github.com/securego/gosec/pull/1149)
-
-> G115: Potential integer overflow when converting between integer types
-
-This issue was way more complex than expected, and required multiple fixes.
+This library is made to help converting any number to another and report an error when it fails
```

Comment:
... if there would be a loss or overflow in the conversion

maybe... though for float that brings up if there is an implicit truncate that is ok or not